prompb/remote.proto

```diff
@@ -35,6 +35,7 @@ message Query {
   int64 start_timestamp_ms = 1;
   int64 end_timestamp_ms = 2;
   repeated prometheus.LabelMatcher matchers = 3;
+  prometheus.ReadParams params = 4;
```

Comment:
Hints would be clearer than params I think

Reply:
I modeled this on `SelectParams`, but I agree that Hints is clearer; changed. 
(we should maybe also rename `SelectParams` -> `SelectHints`)